### ISSUE TITLE
App crash - Share AccountFragment

### ISSUE THREAD

**Issue by code-dagger** [openMF/self-service-app#103]
**Summary:**
Application crashes when a list item in Share Tab clicked. 

**Steps to reproduce:**
How can we reproduce the issue?

**Expected behavior:**
It should display the user detail screen of account info.

**Observed behavior:**
When we click an item it triggers onItemClick method, in which the actual crash is happening.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Intent.toString()' on a null object reference
                      at android.app.Instrumentation.execStartActivity(Instrumentation.java:1485)
                      at android.app.Activity.startActivityForResult(Activity.java:3978)
                      at android.support.v4.app.BaseFragmentActivityJB.startActivityForResult(BaseFragmentActivityJB.java:48)
                      at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:75)
                      at android.support.v4.app.ActivityCompatJB.startActivityForResult(ActivityCompatJB.java:31)
                      at android.support.v4.app.ActivityCompat.startActivityForResult(ActivityCompat.java:169)
                      at android.support.v4.app.FragmentActivity.startActivityFromFragment(FragmentActivity.java:929)
                      at android.support.v4.app.FragmentActivity$HostCallbacks.onStartActivityFromFragment(FragmentActivity.java:1043)
                      at android.support.v4.app.Fragment.startActivity(Fragment.java:932)
                      at android.support.v4.app.Fragment.startActivity(Fragment.java:921)
                      at org.mifos.selfserviceapp.ui.fragments.AccountsFragment.onItemClick(AccountsFragment.java:223)
                      at org.mifos.selfserviceapp.utils.RecyclerItemClickListener$GestureListener.onSingleTapUp(RecyclerItemClickListener.java:103)
                      at android.view.GestureDetector.onTouchEvent(GestureDetector.java:635)
                      at org.mifos.selfserviceapp.utils.RecyclerItemClickListener.onInterceptTouchEvent(RecyclerItemClickListener.java:35)
                      at android.support.v7.widget.RecyclerView.dispatchOnItemTouch(RecyclerView.java:2479)
                      at android.support.v7.widget.RecyclerView.onTouchEvent(RecyclerView.java:2607)
                      at android.view.View.dispatchTouchEvent(View.java:9306)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2548)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2241)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2554)
                      at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2255)
                      at com.android.internal.policy.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2403)
                      at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1737)
                      at android.app.Activity.dispatchTouchEvent(Activity.java:2820)
                      at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:63)
                  	at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.j
```

**Device and Android version:**
Redmi Note 3 - Android 6.0

**Screenshots:**
![screenshot_2017-03-02-12-42-00-738_org mifos selfserviceapp](https://cloud.githubusercontent.com/assets/16591761/23497680/56460bc6-ff4a-11e6-9e37-53b58922e681.png)

**Comment by therajanmaurya** [comment]
Go ahead fix the bug.

**Comment by code-dagger** [comment]
@therajanmaurya bug fix PR #121

**Comment by therajanmaurya** [comment]
fixed in #121
